molgenis-metadata-manager/src/main/frontend/src/components/MetadataManagerEntityEditForm.vue

```diff
@@ -131,8 +131,6 @@
       saveEntityType () {
         if (this.editorEntityType.idAttribute === null || this.editorEntityType.idAttribute === undefined) {
           this.$store.commit(CREATE_ALERT, {type: 'warning', message: 'ID attribute can not be empty'})
-        } else if (this.editorEntityType.labelAttribute === null || this.editorEntityType.labelAttribute === undefined) {
```

Comment:
If the idAttribute is hidden, this warning is a feature and it'd be nice if it still is shown then.